server/src/main/java/org/elasticsearch/search/fetch/subphase/highlight/FastVectorHighlighter.java

```diff
@@ -87,126 +86,119 @@ public HighlightField highlight(FieldHighlightContext fieldContext) {
             hitContext.cache().put(CACHE_KEY, new HighlighterEntry());
         }
         HighlighterEntry cache = (HighlighterEntry) hitContext.cache().get(CACHE_KEY);
-
-        try {
-            FieldHighlightEntry entry = cache.fields.get(fieldType);
-            if (entry == null) {
-                FragListBuilder fragListBuilder;
-                BaseFragmentsBuilder fragmentsBuilder;
-
-                final BoundaryScanner boundaryScanner = getBoundaryScanner(field);
-                if (field.fieldOptions().numberOfFragments() == 0) {
-                    fragListBuilder = new SingleFragListBuilder();
-
+        FieldHighlightEntry entry = cache.fields.get(fieldType);
```

Comment:
Double-checking that none of this logic changed, you just removed the try/ catch? The git diff is probably just unintuitive.

Reply:
Yes, that's correct.